index.js

```diff
@@ -1 +1,37 @@
-console.log("Hello World!");
+const distanceFunctions = require('./distance_calculation.js');
+const usersFunctions = require('./users_operation.js');
+const officeCordsLat = 42.6665921;
+const officeCordsLon = 23.351723;
+const distanceinKmRequiredForInvitation = 100.0;
+
+const fs = require("fs");
+const readline = require("readline");
+
+let usersAndTheirDistanceFromOffice = [];
+let usersFinalListForInvitation = [];
+
+
+const readInterface = readline.createInterface({
+  input: fs.createReadStream("partners.txt"),
+  output: process.stdout,
+  console: false,
+});
+
+// calling the distance calculating function and pushing the user, id and his distance to a new list
+readInterface.on("line", function (line) {
+  let user = JSON.parse(line);
```

Comment:
You can use object destructing here (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment):
```
  let { latitude, longitude, name, partner_id } = JSON.parse(line);
  let distance = distanceFunctions.calculateGreatCircleDistance(
    officeCordsLat,
    officeCordsLon,
    parseFloat(latitude),
    parseFloat(longitude),
  );
  usersAndTheirDistanceFromOffice.push({ Name: name, id: partner_id, Distance: distance });
```